csrc/utils.h

```diff
@@ -60,6 +60,8 @@ bool is_zero_sized_tensor(const std::shared_ptr<c10::TensorType>& tensor_type);
 bool is_cpu_scalar(const at::Tensor& tensor);
 bool is_cpu_scalar(const c10::TensorType& tensor_type);
 
+bool is_meta_scalar(const at::Tensor& tensor);
```

Comment:
A short comment here might be helpful as it is not immediately obvious what a "meta scalar" is.

Reply:
Feel free to ignore this, as it's essentially just a scalar `at::Tensor` that `.is_meta()` :-)